src/components/NurseryWithdrawals/NurseryWithdrawalsTabContent.tsx

```diff
@@ -0,0 +1,23 @@
+import { Typography, useTheme } from '@mui/material';
+import strings from 'src/strings';
+import Card from 'src/components/common/Card';
+import NurseryWithdrawalsTable from './NurseryWithdrawalsTable';
+
+export default function NurseryWithdrawalsTabContent(): JSX.Element {
```

Comment:
styling of title, missing description etc are out of scope of this PR, we can followup later